src/main/scala/uk/gov/nationalarchives/tdr/api/utils/NaturalSorting.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.nationalarchives.tdr.api.utils
+
+import scala.util.matching.Regex
+import java.text.Normalizer
+
+object NaturalSorting {
+  implicit object ArrayOrdering extends Ordering[Array[String]] { // 4
+    val INT: Regex = "([0-9]+)".r
+    def compare(a: Array[String], b: Array[String]) : Int = {
+      val l = Math.min(a.length, b.length)
+      (0 until l).segmentLength(i => a(i) equals b(i)) match {
+        case i if i == l => Math.signum(b.length - a.length).toInt
```

Comment:
This line means that if you have fileName fileName2 fileName10 fileName3, it will sort them as fileName2 fileName3 fileName10 fileName. 
Most of the natural ordering sites online as well as using ls -v will put fileName first which makes sense I think. You can get it to sort like that just by swapping this around so it's `Math.signum(a.length - b.length).toInt`

Reply:
Yeah, right that's how it displays on the file explorer as well. I have swapped the variables as you suggested and it works well. Thanks.